framework/src/modules/dpos_v2/schemas.ts

```diff
@@ -535,3 +535,123 @@ export const genesisStoreSchema = {
 		},
 	},
 };
+
+const delegateJSONSchema = {
+	type: 'object',
+	required: [
+		'name',
+		'totalVotesReceived',
+		'selfVotes',
+		'lastGeneratedHeight',
+		'isBanned',
+		'pomHeights',
+		'consecutiveMissedBlocks',
+	],
+	properties: {
+		name: {
+			type: 'string',
+		},
+		totalVotesReceived: {
+			type: 'string',
+			format: 'uint64',
+		},
+		selfVotes: {
+			type: 'string',
+			format: 'uint64',
+		},
+		lastGeneratedHeight: {
+			type: 'integer',
+			format: 'uint32',
+		},
+		isBanned: {
+			type: 'boolean',
+		},
+		pomHeights: {
+			type: 'array',
+			items: { type: 'integer', format: 'uint32' },
+		},
+		consecutiveMissedBlocks: {
+			type: 'integer',
+			format: 'uint32',
+		},
+	},
+};
+
+export const getDelegateRequestSchema = {
+	$id: 'modules/dpos/endpoint/getDelegateRequest',
+	type: 'object',
+	required: ['address'],
+	properties: {
+		address: {
+			type: 'string',
+			format: 'hex',
```

Comment:
Again, using this capabilities of AJV may not be aligned with our standards and may make "isHexString" function in validator library obsolote. Either let's introduce these capabilities in an informational LIP or add type predicate to "isHexString". If we go to the route of using AJV in a complex way soon everywhere can be "oneOf", "anyOf", we must have specs.

Reply:
Same as above. For JSON validation, standard JSON schema is used